docusaurus.config.ts

```diff
@@ -144,6 +150,10 @@ const config: Config = {
               label: "Download",
               to: "/docs/start/download",
             },
+            {
+              label: "Examples",
+              to: "/docs/examples",
```

Comment:
why not docs/start/examples ? would align with the existing

Reply:
I do not know why, why do we use `start` ?

Reply:
I started it like this as initially I had more files in examples folder than just this one.
Keeping some modularization in case we expand with extra guides. similar could be done to download.
We can keep adding `10.0.0_download`, `10.1.0_download` and that way the folders I think are good to keep.

Simiarly for docs, see the other PR, `documentation.md` for main, `10.0.0_documenation.md` for specific release, we can also include Release notes this way and similar. So from my POV it made sense to separate it in folders.